src/canvas-view/canvas/node-toolbar.component.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .component('nodeToolbar', {
+      templateUrl: 'canvas-view/canvas/node-toolbar.html',
+      bindings: {
+        node: '=',
+        nodeActions: '=',
+        nodeClickHandler: '=',
+        nodeCloseHandler: '='
```

Comment:
We should add comments here (since there is no doc for this) as to what the callback arguments should be.